dev/generate_bom.py

```diff
@@ -49,12 +49,13 @@
 GOOGLE_CONTAINER_BUILDER_SERVICE_BASE_CONFIG = {
   'steps': [
     {
-      'name': 'spinnakerrelease/gradle_cache',
-      'env': ['GRADLE_USER_HOME=/gradle_cache/.gradle'],
+      'name': '',
```

Comment:
This seems like an awkward definition now.
It might be cleaner to instead make this a factory function that returns the configuration dictionary and takes an options parameter. Then instead of fixing it up later (~#190), call the factory function which outright constructs the dictionary with the values you are poking into it below. I'd consider the same for the other config too for consistency even if it isnt as parameterized.